app.yaml

```diff
@@ -2,15 +2,109 @@ runtime: nodejs16
 default_expiration: "2m"
 
 handlers:
-- url: /(.*\.(css|eot|gz|html|ico|jpg|jpeg|js|json|map|png|svg|otf|ttf|woff|woff2))
+
+# font format
+- url: /(.*\.(woff|woff2))
+  mime_type: application/x-font-woff
+  static_files: build/\1
+  upload: build/(.*\.(woff|woff2))
+  secure: always
+  http_headers:
+    X-Content-Type-Options: "nosniff"
```

Comment:
Had to copy/paste two keys (`secure` and `http_headers`) for all `url`. I wish there is an easy way where just set it once.